owl/EASE-OBJ.owl

```diff
@@ -1876,6 +1876,16 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Geometry -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Geometry">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Intrinsic"/>
+        <rdfs:comment>Geometry is a branch of mathematics concerned with questions of shape, size, relative position of figures, and the properties of space.
+https://en.wikipedia.org/wiki/Geometry</rdfs:comment>
```

Comment:
We do not usually add links in the comment.
There are special annotation properties to refer to URLs that should be rather used then including a link in the comment text.

In this case, wikipedia has a citation for their definition. I would also prefer to refer to this.

In comments in DUL, there is often some information about different ontological perspectives on the term, sometimes with references to Literature.

Reply:
Done. Now reference urls are mentioned with SeeAlso tag.